src/R/Editor/Impl/Completions/Providers/RoxygenTagCompletionProvider.cs

```diff
@@ -23,13 +24,13 @@ internal sealed class RoxygenTagCompletionProvider : IRCompletionListProvider {
             var line = context.EditorBuffer.CurrentSnapshot.GetLineFromPosition(context.Position);
             var rawLineText = line.GetText();
             var lineText = rawLineText.TrimStart();
+            var positionInLine = context.Position - line.Start;
+            var typedText = lineText.TextBeforePosition(positionInLine);
 
-            // Check that we are inside the Roxygen comment
-            if (!lineText.StartsWith("#'") || context.Position < rawLineText.Length - lineText.Length + 2) {
-                return completions;
+            // Check that we are inside the Roxygen comment AND just typed character is @
+            if (lineText.StartsWith("#'") && positionInLine >= 2 && typedText.StartsWithOrdinal("@")) {
+                completions.AddRange(RoxygenKeywords.Keywords.Select(k => new EditorCompletionEntry(k, k, string.Empty, _glyph)));
```

Comment:
```cs
RoxygenKeywords.Keywords.Select(k => new EditorCompletionEntry(k, k, string.Empty, _glyph))
```

`RoxygenKeywords.Keywords` is a predefined list. Do you want to save the whole expression as lazy field?

Reply:
Sure